pkg/email/send_email.go

```diff
@@ -29,7 +31,13 @@ type Params struct {
 	LakefsBaseURL      string
 }
 
-func NewEmailer(p Params) *Emailer {
+func NewEmailer(p Params) (*Emailer, error) {
+	if p.SMTPHost != "" {
+		_, err := mail.ParseAddress(p.Sender)
+		if err != nil {
+			return nil, err
```

Comment:
Use `fmt.Errorf` and format an error with more information - the missing data is that we validate 'sender'.
Imagine what will be the error from parse address and the fact that you need to understand what went wrong.

Reply:
Done